### PR TITLE
SageMaker Debugger with XGBoost - US Census data

### PR DESCRIPTION
**Pull request by iyerr3**
*Description of changes:
This notebook demonstrates how to use Amazon SageMaker Debugger to capture the feature importance and SHAP values for an XGBoost model. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: sagemaker-debugger/xgboost_census_explanations/xgboost-census-debugger-rules.ipynb [hunk 1]
```diff
@@ -0,0 +1,955 @@
+{
```
Comment (Vikas-kum) [inline sagemaker-debugger/xgboost_census_explanations/xgboost-census-debugger-rules.ipynb:1]: <p>I think we can do these install when starting with analysis. Few less steps before getting started.</p><p>Initial training start should not require this install. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (Vikas-kum) [inline sagemaker-debugger/xgboost_census_explanations/xgboost-census-debugger-rules.ipynb:1]: <p>formatting looks weird. especially quotes with python.</p><p>/s/<code>metircs/metrics</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

**Comment by Vikas-kum** [comment]
QQ. Have you ran this notebook as sagemaker notebook

**Comment by Vikas-kum** [comment]
Looks good. 
I think sagemaker has a issue, it should start TJ and PJ at same time. 
I can see we are able to fetch tensors earlier than Processing job is getting started during manual analysis phase.

File: sagemaker-debugger/xgboost_census_explanations/xgboost-census-debugger-rules.ipynb (hunk 1 above)
Comment (Vikas-kum) [reply sagemaker-debugger/xgboost_census_explanations/xgboost-census-debugger-rules.ipynb:1]: may be just move smdebug install to later part of script where we do analysis.

**Comment by iyerr3** [comment]
Merging after approval from @Vikas-kum

**Pull request merged**